### ISSUE TITLE
numba --annotate-html fails because template.html is not included in package

### ISSUE THREAD

**Issue by Juanlu001** [numba/numba#1070]
Tested with latest numba 0.18.1:

```
$ numba --annotate-html iod.html iod.py 
Traceback (most recent call last):
  File "iod.py", line 8, in <module>
    from poliastro.util import dot
  File "/home/juanlu/Development/Python/poliastro/poliastro/util.py", line 81, in <module>
    @numba.njit('f8(f8[:], f8[:])')
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/decorators.py", line 156, in wrapper
    disp.compile(sig)
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/dispatcher.py", line 305, in compile
    flags=flags, locals=self.locals)
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 563, in compile_extra
    return pipeline.compile_extra(func)
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 301, in compile_extra
    return self.compile_bytecode(bc, func_attr=self.func_attr)
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 310, in compile_bytecode
    return self._compile_bytecode()
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 550, in _compile_bytecode
    return pm.run(self.status)
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 199, in run
    raise patched_exception
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 191, in run
    res = stage()
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/compiler.py", line 425, in stage_annotate_type
    self.type_annotation.html_annotate()
  File "/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/annotations/type_annotations.py", line 254, in html_annotate
    with open(template_filename, 'r') as template:
FileNotFoundError: [Errno 2] No such file or directory: '/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/annotations/template.html'
```

These are the contents of the corresponding directory:

```
$ tree ~/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/annotations/
/home/juanlu/.miniconda3/envs/poliastro34/lib/python3.4/site-packages/numba/annotations/
├── __init__.py
├── __pycache__
│   ├── __init__.cpython-34.pyc
│   └── type_annotations.cpython-34.pyc
└── type_annotations.py

1 directory, 4 files
```

I don't see `template.html` either in `setup.py` or `MANIFEST.in` so I understand that it is a packaging mistake.

**Comment by seibert** [comment]
Argh!  Good catch.  We'll fix this on Monday and push out a 0.18.2 release.

**Comment by Juanlu001** [comment]
Closed by #1073
